hooks/checkout

```diff
@@ -327,7 +368,14 @@ setup_git_lfs() {
   if [[ "${CANVA_REPO}" =~ .*github\.com.* ]]; then
     # Workaround for GitHub git-lfs API rate limit error:
     # https://github.com/git-lfs/git-lfs/issues/2133#issuecomment-292557138
-    git config "lfs.https://github.com/${CANVA_REPO#*@github.com:}/info/lfs.access" basic
+    # Also setting both https and http URLs, so that if git-proxy requires http URL as remote it should still work.
+    if [[ "${CANVA_REPO}" =~ ^https?://.* ]]; then
+      git config "lfs.https://github.com/${CANVA_REPO#*@github.com/}/info/lfs.access" basic
+      git config "lfs.http://github.com/${CANVA_REPO#*@github.com/}/info/lfs.access" basic
+    else
+      git config "lfs.https://github.com/${CANVA_REPO#*@github.com:}/info/lfs.access" basic
+      git config "lfs.http://github.com/${CANVA_REPO#*@github.com:}/info/lfs.access" basic
+    fi
```

Comment:
I think this is ok, I'm not sure if we need to set the token here, but from a quick look it seems it will take authentication from the origin